TransbankPosSDK/POSAutoservicio.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+using Transbank.Utils;
+using Transbank.Exceptions.CommonExceptions;
+using System.Text;
+using Transbank.Responses.CommonResponses;
+using Transbank.Responses.AutoservicioResponse;
+using Transbank.Exceptions.AutoservicioExceptions;
+using System.Threading.Tasks;
+
+namespace Transbank.POSAutoservicio
+{
+    public class POSAutoservicio : Serial
+    {
+        public POSAutoservicio()
+        {
+
+        }
+
+        public static POSAutoservicio Instance { get; } = new POSAutoservicio();
+
+        public async Task<bool> Poll()
+        {
+            DiscardBuffer();
+
+            if (CantWrite())
+            {
+                throw new TransbankException($"Unable to Poll port {Port.PortName} is closed");
+            }
+
+            try
+            {
+                byte[] buffer = new byte[1];
+                string command = "0100";
+
+                Console.WriteLine($"Out (Hex): {ToHexString(command)}");
+                Console.WriteLine($"Out (ASCII): {command}");
```

Comment:
Maybe this calls can be wrapped on a Logger class, so we can activate/deactivate logs to console?

Reply:
The messages are lost depending on how the SDK is implemented, this is more for what Elías wants